### PR TITLE
Implement statistics on binary classification.

### PR DESCRIPTION
**Pull request by rleonid**

### CODE REVIEW

**Comment by rleonid** [comment]
@smondet If you get a chance, can you please review?

File: src/lib/classify.ml [hunk 1]
```diff
@@ -186,3 +186,96 @@ let gauss_estimate ?(classes=[]) data =
...
+
+module BinaryClassificationPerformance = struct
+
+  type t =
+    | TruePositive
+    | FalseNegative
+    | FalsePositive
+    | TrueNegative
+
+  let datum_to_t d =
```
Comment (smondet) [inline src/lib/classify.ml:215]: Usually those functions are called `of_*`.

**Comment by smondet** [comment]
LGTM
- the record formatting with the leading `;` still hurts my eyes :)
- there is a mix of `let ... in` and `let  ... and ... in` that does not seem very consistent, like there:
  https://github.com/rleonid/oml/pull/48/files#diff-37f52c1d33cfc0b295754aeab9c0e584R264 (the `and` keyword makes me look for mutually recursive things; it's distracting, better not use it if useless)

**Comment by rleonid** [comment]
I tend to use `and` as sort of a you can do all of these bindings at once logic, since you have to write less `in`'s. But that's probably not standard, I'll stop.

**Pull request merged**